app/components/announcement_banner/announcement_banner.js

```diff
@@ -58,24 +61,25 @@ export default class AnnouncementBanner extends PureComponent {
     };
 
     handlePress = () => {
-        const {formatMessage} = this.context.intl;
-        const options = [{
-            text: formatMessage({id: 'mobile.announcement_banner.ok', defaultMessage: 'OK'}),
-        }];
-
-        if (this.props.allowDismissal) {
-            options.push({
-                text: formatMessage({id: 'mobile.announcement_banner.dismiss', defaultMessage: 'Dismiss'}),
-                onPress: this.handleDismiss,
-            });
-        }
-
-        Alert.alert(
-            formatMessage({id: 'mobile.announcement_banner.title', defaultMessage: 'Announcement'}),
-            this.props.bannerText,
-            options,
-            {cancelable: false}
-        );
+        const {navigator, theme} = this.props;
+
+        // TODO figure out how to dismiss banner
+
```

Comment:
Yes without it we’re going to get yelled at, sonwe better figure it out before merging this

Reply:
Talking to @esethna about it.

Reply:
I should've clarified that I marked this as PM review since I was waiting on that design (although I now realize that was a poor way to do that). That's added now and I'll post a screenshot below